### PR TITLE
Higher L2 fees

### PR DESCRIPTION
**Pull request by borislav-itskov**
An attempt to better the UX by increasing the fees a bit on L2s and to overcome the transaction underpriced problem

### CODE REVIEW

File: src/services/bundlers/bundler.ts [hunk 1]
```diff
@@ -192,20 +192,33 @@ export class Bundler {
...
+    // set bigger gas prices
+    // slow: increase by 5%
+    // medium: increase by 6.67%
+    // fast: increase by 10%
+    // ape: increase by 20%
+    const gasPrices: any = {}
+    Object.keys(results).forEach((key: any, index: number) => {
+      const up = BigInt((4 - index) * 5)
+
+      const finalKey = key !== 'standard' ? key : 'medium'
```
Comment (Ivshti) [inline src/services/bundlers/bundler.ts:209]: wouldn't it be a lot more readable if you retain the original code but just use a wrapper function

eg
```
return {
  slow: toBeHex(addExtra(results.max))
  ...
}
```

File: src/consts/networks.ts [hunk 2]
```diff
@@ -65,7 +65,11 @@ const networks: Network[] = [
     feeOptions: {
       is1559: true,
       elasticityMultiplier: 6n,
-      baseFeeMaxChangeDenominator: 50n
+      baseFeeMaxChangeDenominator: 50n,
+      feeIncrease: 50n,
```
Comment (Ivshti) [inline src/consts/networks.ts:69]: what unit is this in?

File: src/consts/networks.ts [hunk 3]
```diff
@@ -117,7 +121,8 @@ const networks: Network[] = [
     features: [],
     feeOptions: {
       is1559: true,
-      minBaseFee: 100000000n // 1 gwei
+      minBaseFee: 100000000n, // 1 gwei
```
Comment (Ivshti) [inline src/consts/networks.ts:124]: can we add comments to explain why those increases are required

File: src/interfaces/network.ts [hunk 4]
```diff
@@ -14,6 +14,10 @@ interface FeeOptions {
   // transactions on Base get stuck on slow as we lower the baseFee a lot
   // so we make the minBaseFee the same as the last block one
   minBaseFeeEqualToLastBlock?: boolean
+  userOpFees?: {
+    // all the fees are in percentages
```
Comment (Ivshti) [inline src/interfaces/network.ts:18]: why is this required in FeeOptions? is it different for each fee option or fixed?

also, using promilles or basis points is more appropriate with BigInts since it allows for more precision

**Review by Ivshti — CHANGES_REQUESTED** [review]

File: src/interfaces/network.ts (hunk 4 above)
Comment (borislav-itskov) [reply src/interfaces/network.ts]: This is removed as discussed offline

**Review by Ivshti — APPROVED** [review]
